From 89bdbb6d6db2a0528b082d7a9698f188f3be5376 Mon Sep 17 00:00:00 2001
From: Intel <intel.com>
Date: Wed, 18 Jun 2025 04:00:38 +0000
Subject: [PATCH] Bump dependency versions for kubevirt v1.5.0

Changes:
- make bump-images
- Update go_version in WORKSPACE from 1.23.4 -> 1.23.10
- Update pinned libvirt/qemu versions in hack/rpm-deps.sh

This patch was created and verified on Kubevirt release v1.5.0
---
 WORKSPACE        | 8 ++++----
 hack/rpm-deps.sh | 6 +++---
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/WORKSPACE b/WORKSPACE
index cf3cdbd89e..ca79740b22 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -171,7 +171,7 @@ load("@bazeldnf//:deps.bzl", "bazeldnf_dependencies", "rpm")
 go_rules_dependencies()
 
 go_register_toolchains(
-    go_version = "1.23.4",
+    go_version = "1.23.10",
     nogo = "@//:nogo_vet",
 )
 
@@ -306,21 +306,21 @@ container_deps()
 # Pull go_image_base
 container_pull(
     name = "go_image_base",
-    digest = "sha256:a7af3ef5d69f6534ba0492cc7d6b8fbcffddcb02511b45becc2fac752f907584",
+    digest = "sha256:b14f0d621bdfd1c967bca28f28ae7c1191e216ce0f34977c9f1e1f5081aae047",
     registry = "gcr.io",
     repository = "distroless/base-debian12",
 )
 
 container_pull(
     name = "go_image_base_aarch64",
-    digest = "sha256:198302a46cd40ab2e24ee54d39ba0919a431e59289fd7b87f798b62e2076c62a",
+    digest = "sha256:4b66c135f2d73c969783fcb918e3b224ea66dac43ce8d2bdd166f362d5dd248c",
     registry = "gcr.io",
     repository = "distroless/base-debian12",
 )
 
 container_pull(
     name = "go_image_base_s390x",
-    digest = "sha256:642791d0afe3d071e365923e65203074f30bad4ca621309d2eab52bf2d32077e",
+    digest = "sha256:fa81a9ab9966083922a8465506accd01cad4ebb787f7e11309d464e19b94d097",
     registry = "gcr.io",
     repository = "distroless/base-debian12",
 )
diff --git a/hack/rpm-deps.sh b/hack/rpm-deps.sh
index c3da8cfda5..16784861e6 100755
--- a/hack/rpm-deps.sh
+++ b/hack/rpm-deps.sh
@@ -6,11 +6,11 @@ source hack/common.sh
 source hack/bootstrap.sh
 source hack/config.sh
 
-LIBVIRT_VERSION=${LIBVIRT_VERSION:-0:10.10.0-4.el9}
-QEMU_VERSION=${QEMU_VERSION:-17:9.1.0-12.el9}
+LIBVIRT_VERSION=${LIBVIRT_VERSION:-0:11.10.0-4.el9}
+QEMU_VERSION=${QEMU_VERSION:-17:10.1.0-10.el9} 
 SEABIOS_VERSION=${SEABIOS_VERSION:-0:1.16.3-4.el9}
-EDK2_VERSION=${EDK2_VERSION:-0:20241117-2.el9}
+EDK2_VERSION=${EDK2_VERSION:-0:20241117-8.el9}
-LIBGUESTFS_VERSION=${LIBGUESTFS_VERSION:-1:1.54.0-3.el9}
+LIBGUESTFS_VERSION=${LIBGUESTFS_VERSION:-1:1.54.0-11.el9}
 GUESTFSTOOLS_VERSION=${GUESTFSTOOLS_VERSION:-0:1.52.2-2.el9}
-PASST_VERSION=${PASST_VERSION:-0:0^20250121.g4f2c8e7-3.el9}
+PASST_VERSION=${PASST_VERSION:-0:0^20251210.gd04c480-2.el9}
 VIRTIOFSD_VERSION=${VIRTIOFSD_VERSION:-0:1.13.0-1.el9}
-- 
2.43.0

